libs/dataset-explorer/src/lib/DataAnalysisTab.tsx

```diff
@@ -15,7 +15,7 @@ import React from "react";
 import { dataAnalysisTabStyles } from "./DataAnalysisTab.styles";
 import { DataBalanceTab } from "./DataBalanceTab";
 import { DatasetExplorerTab } from "./DatasetExplorerTab";
-import { TableView } from "./TableView/TableView";
+import { TableViewTab } from "./TableView/TableViewTab";
```

Comment:
Was this change reviewed by PM and designer? Does it need to be changed in few days/months for big data changes?

Reply:
This change was reviewed by Mehrnoosh and Jingya.  I'm not sure about how this affects big data.  I think we have a different PM and designer working on the big data UI?

Reply:
For big data changes, I meant there are few changes wrt table view in big data. So I am just concerned if this needs to be revisited when working on big data changes.

Reply:
It might be, we can revisit once we get to it